releases/2022-08-26-releasenotes.md

```diff
@@ -0,0 +1,120 @@
+
+# New Authorization Options
+Vendia Share now supports [multiple forms of authorization](https://www.vendia.net/docs/share/node-authentication) to a Node within a Uni.  
+
+## API Keys
+For those who prefer to create and use API Keys for authorization, this can now be performed through the [Share UI](https://www.vendia.net/docs/share/node-authentication#creating-an-api-key-in-vendia), the [Share GraphQL API](https://www.vendia.net/docs/share/node-authentication#creating-an-api-key-via-graphql-mutation), or the [Share CLI](https://www.vendia.net/docs/share/node-authentication#creating-an-api-key-via-the-cli).  A Node can now support multiple API Keys and each key can have a unique expiration time.
+
+```shell
+share node add-api-key \
+    --uni 'my-uni-name' \
+    --node 'my-node-name' \
+    --name 'my-api-key-name' \
+    --expiry '2022-08-31'
+```
+
+When using an API Key, place the value of the key in the `Authorization` header when submitting GraphQL requests to the Node to which the API Key is associated.
+
+```shell
+curl 'https://xxxxxxx.execute-api.us-west-2.amazonaws.com/graphql/' \
+    -H 'Authorization: MY_API_KEY_HERE' \
+    -H 'content-type: application/json' \
+    -H 'accept: application/json' \
+    --data-raw '{"query":"query blocksQuery {  listVendia_BlockItems {    Vendia_BlockItems {      blockId      blockHash    }  }}","variables":null,"operationName":"blocksQuery"}' \
+    --compressed
+```
+
+## JWTs
+For those who prefer to use [JSON Web Tokens (JWTs)](https://jwt.io/) for authorization, this can now be configured through the [Share UI](https://www.vendia.net/docs/share/node-authentication#creating-jwt-authentication-in-vendia) or the [Share GraphQL API](https://www.vendia.net/docs/share/node-authentication#create-jwt-authentication-via-graphql).  JWT authorization requires a small handful of configuration properties, from which your Vendia Share Node can perform all the necessary JWT verifications needed on subsequent GraphQL API calls.
+
+When using a JWT, place the value of the key in the `Authorization` header when submitting GraphQL requests to the Node whose configuration reflects the identity provider that issued the JWT.  Note the `Bearer` prefix in the `Authorization` header value below.
+
+```shell
+curl 'https://xxxxxxx.execute-api.us-west-2.amazonaws.com/graphql/' \
+    -H 'Authorization: Bearer MY_JWT_HERE' \
+    -H 'content-type: application/json' \
+    -H 'accept: application/json' \
+    --data-raw '{"query":"query blocksQuery {  listVendia_BlockItems {    Vendia_BlockItems {      blockId      blockHash    }  }}","variables":null,"operationName":"blocksQuery"}' \
+    --compressed
+```
+
+With the release of these new authorization options comes the end of the legacy authorization options previously supported.    
+
+# Sorting Results
+The ability to sort results based on a field was a frequent request from our users over the last few quarters.  We heard you and we did it!
```

Comment:
Thoughts about: "We heard you and we did it" --> "We ranked this feedback in ascending order!"

Reply:
😆

Added a variant of this.